libraries/azurerm_storage_account.rb

```diff
@@ -30,7 +30,7 @@ def initialize(name: 'default', resource_group: nil)
     @exists = true
   end
 
-  def have_recently_generated_access_key
+  def has_recently_generated_access_key?
```

Comment:
Good catch!

Reply:
Was kind of shoe horned after the fact but yes 😅